src/validators.ts

```diff
@@ -166,3 +169,16 @@ export const array = <U extends readonly AnySchema[]>(...arr: readonly [...U]) =
     U
   >;
 };
+
+export const any = () => {
+  return {
+    [TYPEOFWEB_SCHEMA]: true,
+    __validator: ANY_VALIDATOR,
+    __modifiers: { optional: true, nullable: true },
```

Comment:
Jest to potrzebne? W validate check czy jest `any` i tak wjeżdża pierwszy. I jak teraz wygląda typ otrzymywany? `any | undefined | null`, czy się mylę?

Reply:
Mam na myśli, że chyba powinno być
```suggestion
    __modifiers: { optional: false, nullable: false },
```

Reply:
`unknown`